src/ebonite/ext/onnx/model.py

```diff
@@ -0,0 +1,107 @@
+import contextlib
+import os
+from abc import abstractmethod
+from typing import Dict, List, Type, Union
+
+import onnx
+from onnx import ModelProto
+from pyjackson.decorators import cached_property
+
+from ebonite.core.analyzer import TypeHookMixin
+from ebonite.core.analyzer.model import BindingModelHook
+from ebonite.core.objects import ModelIO, ModelWrapper, Requirements
+from ebonite.core.objects.artifacts import Blobs, InMemoryBlob
+from ebonite.core.objects.wrapper import FilesContextManager
+from ebonite.utils.abc_utils import is_abstract_method
+from ebonite.utils.importing import module_importable
+
+_DEFAULT_BACKEND = 'onnxruntime'
+
+
+def set_default_onnx_backend(backend: Union[Type['ONNXInferenceBackend'], str]):
+    global _DEFAULT_BACKEND
+    if not isinstance(backend, str):
+        backend = backend.name
+    if backend not in ONNXInferenceBackend.subtypes:
+        raise ValueError(f'unknown onnx backend {backend}')
+    _DEFAULT_BACKEND = backend
+
+
+class ONNXModelIO(ModelIO):
+    FILENAME = 'model.onnx'
+
+    @contextlib.contextmanager
+    def dump(self, model: ModelProto) -> FilesContextManager:
+        yield Blobs({self.FILENAME: InMemoryBlob(model.SerializeToString())})  # TODO change to LazyBlob
```

Comment:
todo?

Reply:
it will use code from datasets branch, so we fix it after merge